src/main/java/org/isf/patient/manager/PatientBrowserManager.java

```diff
@@ -104,7 +104,7 @@ public Patient savePatient(Patient patient) throws OHServiceException {
 	 * @return the Patient that match specified name (could be null)
 	 * @throws OHServiceException 
 	 * @deprecated use getPatient(Integer code) for one patient or 
-	 * getPatientWithHeightAndWeight(regex) for a list
+	 * getPatientsByOneOfFieldsLike(String regex) for a list
 	 */
 	public Patient getPatientByName(String name) throws OHServiceException {
```

Comment:
Missing `@Deprecated` annotation

Reply:
Yep, it seems not needed any more by the majority of the compilers, but for homogeneity of the code I would add it.

Maybe we can investigate more in order to see if to stay with `@deprecated` in Javadoc only and slim down the code, but it's not a priority of course.

Thanks!

Reply:
I'm not sure but does the Java compiler "read" the JavaDoc?  If you don't add the annotation then the compiler can't flag uses of deprecated methods I believe.

Reply:
well there's a little bit debate on it... https://stackoverflow.com/a/5039811/5309541

on my IDE it is shown as deprecated in any way:

![image](https://user-images.githubusercontent.com/2938553/100229552-b5c75600-2f24-11eb-914e-d310b4f68d50.png)

Reply:
Commited anyway: https://github.com/informatici/openhospital-core/pull/254/commits/aa7eb54519d481bd54ea013ae71954a98e00b399

Note: the commit's comment was correct (Uppercase) but GitHub changed it 😄